### DISCUSSION TITLE
Mesh Points

### DISCUSSION THREAD

**Discussion by PsychoAxeKiller** [jneilliii/OctoPrint-BedLevelVisualizer#384] [Q&A]
Just curious, i can get my ender3v2 to do the standard 3x3 mesh point check ( creality Filmware from website for bl touch)
but using octoprint i was wondering if i can ask octoprint to do a bigger mesh say a 5x5 or a 9x9 to get a finer adjustment but only while using octoprint?
if so what command would i need in the plugin?

thanks for any hekpo

**Comment by jneilliii** [comment]
The plugin only visualizes those mesh reports and sends the gcode to get the report. The number of points probed is dependent on your firmware and leveling type, so to change the number of points typically requires updating the firmware in code and reflashing. There are tons of videos about, including ones from Teaching Tech, that explain this process. You'd be better off replacing the Creality firmware anywhere with mainline Marlin as theirs tends to have a lot of weird things going on from what I've seen on the forum.

https://community.octoprint.org/t/printers-known-to-ship-with-broken-or-unsafe-firmware-or-broken-electronics/25911

**Reply by PsychoAxeKiller** [reply]
thanks, i have just updated to the smith 5x5 fast filmware, its enabled to me get a better UI and a fast 5x5 mesh . which gives more detal:) thanks for the info.
i was hoping i could do it without upgrading but just wanted to check . thankyou
